packages/react-components/src/components/Select/Select.unit.spec.jsx

```diff
@@ -200,7 +200,8 @@ describe('<Select>', () => {
         onValueChange={() => {}}
       />,
     );
-    expect(tree.find('label').text()).to.equal('my label(*Required)');
+    expect(tree.find('label').text()).to.include('my label(*');
```

Comment:
We may want to have the dependency initialized to the `cimode` language in a `before` hook so this is a bit less weird.